app/src/main/java/io/mochahub/powermeter/data/WorkoutEntity.kt

```diff
@@ -0,0 +1,31 @@
+package io.mochahub.powermeter.data
+
+import androidx.room.Entity
+import androidx.room.ForeignKey
+import androidx.room.PrimaryKey
+import java.time.Instant
+import java.util.UUID
+
+// TODO: Add indices
```

Comment:
What is this TODO referring to?

Reply:
Indexing the database will make increase performance. We aren't suffering any performance issues, so we don't really need it right now.